session/session_test.go

```diff
@@ -3145,6 +3145,31 @@ func (s *testSessionSuite2) TestStmtHints(c *C) {
 	c.Assert(tk.Se.GetSessionVars().GetReplicaRead(), Equals, kv.ReplicaReadFollower)
 }
 
+// Test MEMORY_QUOTA hint in POINT_GET/BATCH_POINT_GET
+func (s *testSessionSuite2) TestPointGetStmtHints(c *C) {
```

Comment:
I think `TestPointGetMemoryTracking` or something like that would be more accurate for this issue. Hints is just the method to verify the correctness.

Reply:
done